app/views/settings.js

```diff
@@ -24,7 +24,7 @@
                 notificationSoundMuted: await F.state.get("notificationSoundMuted"),
                 allowBugReporting: !(await F.state.get("disableBugReporting")),
                 allowUsageReporting: !(await F.state.get("disableUsageReporting")),
-                hasPushNotifications: !!(await F.state.get('serverGcmHash')),
+                hasPushNotifications: !!(await F.state.get('serverGcmHash')) || (F.electron && self.Notification && Notification.permission),
```

Comment:
Hmmm..  This isn't exactly true.  Could we just filter the push notification thing out of the template by wrapping it in {{#if isElectron}} .... {{/if}} ?

Reply:
Done and pushed.